docs/classic-ui/images.md

```diff
@@ -272,6 +272,54 @@ The possible options for `mode` are listed below, where the default option is `s
 
     Deprecated option names: `scale-crop-to-fill`, `up`.
 
+### Images Test Rendering
+
+Plone's Classic UI includes special demonstration views that render ready-made examples of image handling and responsive image components using your site's active theme. These views help developers verify image scaling, srcset generation, and explore available image optimization patterns. These views work on image content types (e.g., images or documents with images).
+
+Append the view name `/@@images-test` to an image URL, optionally followed by an anchor like `#srcset` for specific sections.
+
+````{card}
+```{image} /_static/images-test/images-test.png
+:alt: Images Test
+:target: /_static/images-test/images-test.png
+```
++++
+_Images Test_
+````
+Example: `https://classic.demo.plone.org/en/demo/an-image.jpg/@@images-test#srcset`
+
+It displays the following examples (all rendered with the current theme’s styling):
```

Comment:
```suggestion
It displays the following examples (all rendered with the current settings in `/@@imaging-controlpanel`):
```

Reply:
Oh, c'mon! @acsr unresolving this one, too.